addons/pos_sale/static/src/overrides/components/control_buttons/control_buttons.js

```diff
@@ -8,15 +8,18 @@ patch(ControlButtons.prototype, {
     onClickQuotation() {
         this.dialog.add(SelectCreateDialog, {
             resModel: "sale.order",
-            noCreate: true,
             multiSelect: false,
+            noCreate: true,
             domain: [
                 ["state", "!=", "cancel"],
-                ["invoice_status", "!=", "invoiced"],
+                ["amount_unpaid", ">", 0],
```

Comment:
We should not make this change. Think of a pos.order as replacement of an invoice.